codecov_auth/migrations/0034_oktauser.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.2 on 2023-07-25 18:08
+
+import uuid
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("codecov_auth", "0033_sentryuser"),
```

Comment:
You might want to redo this migration since this PR was done a while ago, and we're in migration 0039

Reply:
Oh yeah, thanks - looks like all the tests broke when I merged `main`